docs/Encyclopedia/pages/Biospecimen_Data.md

```diff
@@ -17,7 +17,6 @@ Biospecimen data can be downloaded from the GDC in several formats:
 
 * __API Retrieval:__ Specific information about each biospecimen entity can be queried from the API in a tab-delimited or JSON format. This information can be retrieved programatically.
 * __Biospecimen Supplements:__ Biospecimen supplements are stored in XML format and can be downloaded from the GDC Portal. Supplements may contain biospecimen fields that do not appear in the API as well as those that do. These can include fields that only apply to certain projects or have not yet been incorporated into the GDC Data Dictionary.
-* __Biotab Files:__ Biotab files specific supplemental files that are available in the GDC Legacy Archive as tab-delimited files on a project-level basis. These may also include fields that are not available in the GDC API.

```

Comment:
These were also added to the active portal.  Add this sentence back, remove the references to Legacy, and perhaps add some filter instructions (I think it's best filtered by the data format facet)